changelog.txt

```diff
@@ -1,5 +1,8 @@
 *** WooCommerce Square Changelog ***
 
+= 4.5.1 - TBD =
```

Comment:
@Sidsector9 note that you can stop applying updates in the changelog.txt and readme.txt files for the changelog entries as we can now apply those all at once during the extension release process.